x-pack/plugins/security/public/views/management/roles.html

```diff
@@ -79,19 +79,15 @@
           <div
             class="kuiNoItems"
           >
-            <span
-              i18n-id="xpack.security.management.roles.noFoundMatchingRolesDescription1"
-              i18n-default-message="No "
-              i18n-context="Part of composite label xpack.security.management.roles.noFoundMatchingRolesDescription1 + {matchingText} + xpack.security.management.roles.noFoundMatchingRolesDescription2"
-            ></span>
             <span
               ng-show="query"
```

Comment:
@joshdover, is there any rule according to usage of `ng-show`/ `ng-if` in Kibana? I left `ng-show` here, but I know that sometimes it's preferable to use `ng-if` to remove an element from the DOM.

Reply:
I don't know if we have a rule, but there are some cases where ngIf doesn't work correctly. If you can test that ngIf would work right in this case, then I think it'd be preferable to use that so that this tag doesn't get picked up by screenreaders.

Reply:
Ok. I've checked with `ng-if`, it works as excepted.